### PR TITLE
Enable specs

### PR DESCRIPTION
**Pull request by andrew-codes**
Wallaby
CLI

### CODE REVIEW

**Comment by andrew-codes** [comment]
also other misc. things like editorconfig, and correct server restart on change of server files only

**Comment by andrew-codes** [comment]
Travis integration also to boot!

**Comment by andrew-codes** [comment]
Oh, and devtools for Redux

File: nodemon.json [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
   "ignore": [
-    "*.spec.js"
+    "*.spec.js",
+    "client/**/*",
+    "shared/**/*"
   ],
   "watch": [
-    "src/server"
+    "src/server",
+    "src/shared/store"
```
Comment (walkerrandolphsmith) [inline nodemon.json]: Ah good catch

File: src/server/index.js [hunk 2]
```diff
@@ -1,4 +1 @@
-require('babel-core/register')({
-    extensions: ['.es6', '.es', '.js']
-});
-require('./server');
+import server from './server';
```
Comment (walkerrandolphsmith) [inline src/server/index.js]: Believe this was let over from not using nodemon, but again another good catch! Much cleaner.

File: src/shared/store/index.js [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
+     debugger;
+ }
+    let enhancers = [createMiddlewares(history)];
+    if(env.nodeEnv !== 'production') {
+        enhancers = enhancers.concat([
+            typeof window === 'object' && typeof window.devToolsExtension !== 'undefined' ? window.devToolsExtension() : (f) => f,
+            devTools()
+        ]);
+    }
+    const store = createStore(rootReducer, initialState, compose(...enhancers));
```
Comment (walkerrandolphsmith) [inline src/shared/store/index.js]: Wire up dev tools when not in production. Like how additional middlewares are considered enhancers. Good naming convention!

File: wallaby.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            {pattern: 'node_modules/react/dist/react-with-addons.js', instrument: false},
+            {pattern: 'src/**/*.js', load: false},
+            {pattern: 'src/**/*.spec.js', ignore: true},
+            {pattern: 'src/client/**/*.js', ignore: true}
+        ],
+        tests: [
+            {pattern: 'src/**/*.spec.js', load: false}
+        ],
+        compilers: {
+            '**/*.js': wallaby.compilers.babel(),
```
Comment (walkerrandolphsmith) [inline wallaby.js]: Do we need to pass in our babel config since its not in the `.babelrc` file and instead in `package.json`?

**Pull request merged**